R/data_table_templates.R

```diff
@@ -0,0 +1,105 @@
+## Capture data template
+capture_data_template <- tibble(CaptureID = NA_character_, ## Missing data not allowed
+                                IndvID = NA_character_, ## Missing data not allowed
+                                Species = NA_character_, ## Missing data not allowed
+                                Sex_observed = NA_character_,
+                                BreedingSeason = NA_integer_, ## Missing data not allowed
+                                CaptureDate = as.Date(NA), ## Missing data not allowed
+                                CaptureTime = NA_character_,
+                                ObserverID = NA_character_,
+                                LocationID = NA_character_,
+                                CaptureAlive = NA_complex_,
+                                ReleaseAlive = NA_complex_,
+                                CapturePopID = NA_character_, ## Missing data not allowed
+                                CapturePlot = NA_character_,
+                                ReleasePopID = NA_character_, ## Missing data only allowed if ReleaseAlive is False
+                                ReleasePlot = NA_character_, ## Missing data only allowed if ReleaseAlive is False
+                                Mass = NA_real_,
+                                Tarsus = NA_real_,
+                                OriginalTarsusMethod = NA_character_,
+                                WingLength = NA_real_,
+                                Age_observed = NA_integer_,
+                                Age_calculated = NA_integer_,
+                                ChickAge = NA_integer_,
+                                ExperimentID = NA_character_)
+
+## Save
+save(capture_data_template, file = "./data/Capture_data_template.rda")
+
+########################################################################
+########################################################################
+
+## Individual data template
+individual_data_template <- tibble(IndvID = NA_character_, ## Missing data not allowed
+                                Species = NA_character_, ## Missing data not allowed
+                                PopID = NA_character_, ## Missing data not allowed
+                                BroodIDLaid = NA_character_,
+                                BroodIDFledged = NA_character_,
+                                RingSeason = NA_integer_,
+                                RingAge = NA_character_,
+                                Sex_calculated = NA_character_,
+                                Sex_genetic = NA_character_)
+
+## Save
+save(individual_data_template, file = "./data/Individual_data_template.Rda")
```

Comment:
It's annoying, but R expects .rda (i.e. lowercase) rather than .Rda in the data folder, even though they are functionally the same. 

```suggestion
save(individual_data_template, file = "./data/Individual_data_template.rda")
```